cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -816,6 +847,70 @@ pub mod bridging {
 		}
 	}
 
+	pub mod to_ethereum {
+		use super::*;
+
+		parameter_types! {
+			pub EthereumLocation: MultiLocation = MultiLocation::new(2, X1(GlobalConsensus(EthereumNetwork::get())));
+
+			/// User fee for ERC20 token transfer back to Ethereum.
+			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
+			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
+			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
+			pub const BridgeHubEthereumBaseFeeInROC: u128 = 2_750_872_500_000;
+			pub SiblingBridgeHubWithEthereumInboundQueueInstance: MultiLocation = MultiLocation::new(
+				1,
+				X2(
+					Parachain(SiblingBridgeHubParaId::get()),
+					PalletInstance(snowbridge_rococo_common::INBOUND_QUEUE_MESSAGES_PALLET_INDEX)
+				)
+			);
+
+			/// Set up exporters configuration.
+			/// `Option<MultiAsset>` represents static "base fee" which is used for total delivery fee calculation.
+			pub BridgeTable: sp_std::vec::Vec<NetworkExportTableItem> = sp_std::vec![
+				NetworkExportTableItem::new(
+					EthereumNetwork::get(),
+					Some(sp_std::vec![
+						EthereumLocation::get().interior.split_global().expect("invalid configuration for Ethereum").1,
```

Comment:
I think `EhtereumLocation` const is not really needed, here can be replaced with `X1(GlobalConsensus(EthereumNetwork::get()))` directly.

Anyway, would expect here filter target as some "smart contract accountIId like" and not the whole `GlobalConsensus(EthereumNetwork`? Do we really want to allow here bridging to everywhere to the Ethereum? Maybe, it is covered on BridgeHub, which I haven't reviewed yet.

Reply:
> I think EhtereumLocation const is not really needed, here can be replaced with X1(GlobalConsensus(EthereumNetwork::get())) directly.

Good point, I'll make the change.

> Anyway, would expect here filter target as some "smart contract accountIId like" and not the whole GlobalConsensus(EthereumNetwork?

As far as I remember, we decided in one of the bi-weekly Parity calls that it is OK to route all Ethereum traffic to snowbridge. For future versions, we would like to use a bridge ID for routing and have added a proposal for it: https://github.com/paritytech/xcm-format/pull/49

Reply:
Yes, confirmed, we're taking ownership of the `NetworkId::Ethereum` namespace until something like paritytech/xcm-format#49 is in place

Reply:
ok